### PR TITLE
Update Changelog v0.0.4

### PR DESCRIPTION
**Pull request by buschtoens**
@rlivsey I updated the changelog and the additions / fixes would justify a npm version patch.

```bash
npm version patch && npm publish
```

I'm sure @alekspetrov will be quite happy. :wink:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 
 ## Master
 
+## v0.0.4
+
+* #5 The path to an image can now optionally have a `.svg` extension.
+* #6 The default source directory changed from `./public/images` to `./public`.
+* #7 If there is an error during SVG optimization, the build is now aborted and a descriptive error is displayed.
+* #7 If there are no images present, nothing happens and the build continues. [More info](https://github.com/minutebase/ember-inline-svg/issues/1#issuecomment-70625510).
```
Comment (buschtoens) [inline CHANGELOG.md]: That's no typo.

**Comment by rlivsey** [comment]
Nice one thanks, I left a couple of comments on one of the commits - sorry should have added them to the pull request - once that's sorted I'll get this released!

**Comment by buschtoens** [comment]
No problem. I'll get that sorted out. It was my jshint nagging me. :smile:

**Comment by buschtoens** [comment]
#11 merged.

**Comment by rlivsey** [comment]
Seemed worth a minor bump so I've released it as v0.1.0 - thanks for all the help @buschtoens

**Comment by buschtoens** [comment]
Anytime! :)

**Pull request merged**